scripts/linters/test_files/invalid_style_tag.html

```diff
@@ -0,0 +1,2 @@
+<!--Below line contain style tag which is not allowd-->
```

Comment:
allowed

Reply:
Below line contains a style tag which is not allowed.

Reply:
Done.